uploadNeuroDB/NeuroDB/MRIProcessingUtility.pm

```diff
@@ -58,9 +70,36 @@ sub new {
     $self->{'debug'} = $debug;
     $self->{'TmpDir'} = $TmpDir;
     $self->{'logfile'} = $logfile;
+    $self->{'db'} = $db;
+    
+    $self->{'mriScanTypeOB'} = undef;
+    
     return bless $self, $params;
 }
 
+=pod
+
+=head3 C<getMriScanTypeOB()>
```

Comment:
You are right, it is not used anymore. It was actually used by the chunk of code that was moved to MRI::scan_type_text_to_id(), which explains all the crap code you found earlier in that method. This will be corrected.